### DISCUSSION TITLE
Emoji support for Actions log output

### DISCUSSION THREAD

**Discussion by GitPaulo** [community/community#56570] [Actions] [answered]
### Select Topic Area

Question

### Body

I believe emoji are not supported in Github Action log blocks. This is both for short codes `:example:` and emojis codepoints `👋`.

However, I do know they are supported for topmost category headers. Does anyone know if i'm wrong/there is any work done towards supporting this?

**Examples**

Log blocks are not supported, emojis show as `?`, 

![image](https://github.com/community/community/assets/18451329/af4eece6-04df-43e2-8467-a0d3425da208)

Category headers are supported,

![image](https://github.com/community/community/assets/18451329/be2ec516-38e5-430c-b3c2-2e23539bcdae)
> this image is not mine

**Comment by jge162** [comment, marked as answer]
I do not believe they are supported, characters cannot be displayed properly.
